app/components/ViewContainer.js

```diff
@@ -54,21 +71,26 @@ export default class ViewContainer extends Component {
           targetPin={targetPin}
           setTarget={setTarget}
           /> : void 0 }
-      { this.state.view != 'list' ? <Button
-          onPress={this.toggleView.bind(this, 'list')}
-        >
-          List
-        </Button> : void 0 }
-      { this.state.view != 'ar' ? <Button
-          onPress={this.toggleView.bind(this, 'ar')}
-        >
-          AR
-        </Button> : void 0 }
-      { this.state.view != 'map' ? <Button
+        <View style={styles.ViewMenu}>
+        { this.state.view != 'ar' ? <Button
+            style={styles.ViewButton}
+            onPress={this.toggleView.bind(this, 'ar')}
+          >
+            AR
+          </Button> : void 0 }
+        { this.state.view != 'map' ? <Button
+          style={styles.ViewButton}
           onPress={this.toggleView.bind(this, 'map')}
-        >
+          >
           Map
-        </Button> : void 0 }
+          </Button> : void 0 }
```

Comment:
@asu91 @jake-shasteen @bignell I also noticed that in list view the button kind of blocks the first entry in the list. I'd recommend a header that says "list" or just bump the list down so it doesn't look like there's a big overlapping button blocking this. new issue?